Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:2.7
```

Comment:
i believe so... i have done simple docker deploys on heroku so ill dive into this